console/src/main/java/org/matrix/console/activity/AccountCreationActivity.java

```diff
@@ -35,7 +35,7 @@
     public static String EXTRA_HOME_SERVER_ID = "org.matrix.console.activity.EXTRA_HOME_SERVER_ID";
 
     WebView mWebView = null;
-    String mHomeServer = null;
+    String mHomeServerUrl = null;
```

Comment:
Did you really mean to put this at `package` scope visibility? Missing `private` potentially.